video/cloud-client/analyze/analyze_test.py

```diff
@@ -22,6 +22,9 @@
 
 
 BUCKET = os.environ['CLOUD_STORAGE_BUCKET']
+
+
+# Videos used for these tests can be found in the gs://demomaker/ public bucket
```

Comment:
I'm going to send a PR to update these tests to explicitly use the demomaker bucket to save future contributor friction, so this comment will no longer be necessary. Please remove so we can avoid merge conflicts.